esphome/components/zyaura/zyaura.h

```diff
@@ -0,0 +1,90 @@
+#pragma once
+
+#include "esphome/core/component.h"
+#include "esphome/core/esphal.h"
+#include "esphome/components/sensor/sensor.h"
+
+namespace esphome {
+namespace zyaura {
+
+static const uint8_t ZA_MAX_MS = 2;
+static const uint8_t ZA_MSG_LEN = 5;
+static const uint8_t ZA_FRAME_SIZE = 40;
+static const uint8_t ZA_MSG_DELIMETER = 0x0D;
+
+static const uint8_t ZA_BYTE_TYPE = 0;
+static const uint8_t ZA_BYTE_HIGH = 1;
+static const uint8_t ZA_BYTE_LOW = 2;
+static const uint8_t ZA_BYTE_SUM = 3;
+static const uint8_t ZA_BYTE_END = 4;
+
+typedef enum {
```

Comment:
```suggestion
enum ZaDataType {
```

See google style guide.